src/RunProcessAsTask/ProcessEx.overloads.cs

```diff
@@ -11,16 +9,51 @@ namespace RunProcessAsTask
     // see http://msdn.microsoft.com/en-us/library/system.diagnostics.process.start.aspx
     public partial class ProcessEx
     {
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="fileName">An application or document which starts the process.</param>
         public static Task<ProcessResults> RunAsync(string fileName)
             => RunAsync(new ProcessStartInfo(fileName));
 
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="fileName">An application or document which starts the process.</param>
+        /// <param name="cancellationToken">Notification to cancel this process.</param>
+        public static Task<ProcessResults> RunAsync(string fileName, CancellationToken cancellationToken)
+            => RunAsync(new ProcessStartInfo(fileName), cancellationToken);
+
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="fileName">An application or document which starts the process.</param>
+        /// <param name="arguments">Command-line arguments to pass to the application when the process starts.</param>
         public static Task<ProcessResults> RunAsync(string fileName, string arguments)
             => RunAsync(new ProcessStartInfo(fileName, arguments));
 
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="fileName">An application or document which starts the process.</param>
+        /// <param name="arguments">Command-line arguments to pass to the application when the process starts.</param>
+        /// <param name="cancellationToken">Notification to cancel this process.</param>
+        public static Task<ProcessResults> RunAsync(string fileName, string arguments, CancellationToken cancellationToken)
+            => RunAsync(new ProcessStartInfo(fileName, arguments), cancellationToken);
+
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="processStartInfo">Specifies a set of values that are used when you start a process.</param>
         public static Task<ProcessResults> RunAsync(ProcessStartInfo processStartInfo)
             => RunAsync(processStartInfo, CancellationToken.None);
 
-        public static Task<ProcessResults> RunAsync(ProcessStartInfo processStartInfo, CancellationToken cancellationToken) =>
-            RunAsync(processStartInfo, new List<string>(), new List<string>(), cancellationToken);
+        /// <summary>
+        /// Runs asynchronous process.
+        /// </summary>
+        /// <param name="processStartInfo">Specifies a set of values that are used when you start a process.</param>

```

Comment:
Since we're just passing through to Process.Start, it might make more sense for us to copy the parameter descriptions theirs? For "filename", "arguments", ProcessStartInfo, etc, at least.  For instance looking at https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.start?view=netframework-4.8#System_Diagnostics_Process_Start_System_Diagnostics_ProcessStartInfo_

    The ProcessStartInfo that contains the information that is used to start the process, including the file name and any command-line arguments.

For cancellationToken, we could probably use the description from framework async methods like https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.readasync?view=netframework-4.8

    The token to monitor for cancellation requests.

The things that don't have any mapping in the framework (like the stdout/stderr lists we pass in) we can probably specify whatever makes sense.

FFR, got the ProcessStartInfo param xml from https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Diagnostics/Process.xml

Reply:
@suchoss FWIW I decided to merge this and just make the changes post-merge since there was no real reason you needed to make these edits, I'm happy to make them.

Thanks for the PR!!

Reply:
@jamesmanning I used tooltips from Visual Studio for writing this summary. Thanks and have a nice day ;)